docker-compose.yml

```
@@ -46,11 +46,11 @@ services:
       - DATABASE_URL=ecto://postgres:postgres@db/eve_tracker_dev
       - REDIS_URL=redis://redis:6379
       - PHX_HOST=localhost
-      - PHX_PORT=4000
+      - PHX_PORT=4010
     env_file: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)